ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -197,8 +197,8 @@ public void sendRequest(
       requestBuilder.method(method, multipartBuilder.build());
     } else {
       // Nothing in data payload, at least nothing we could understand anyway.
-      // Ignore and treat it as if it were null.
-      requestBuilder.method(method, null);
+      // Creating empty requestBody to solve issue similar to #3371
```

Comment:
I agree with @lexs 
If you are going to have a comment, a better one is closer to what was there before and say we default to an empty body